core/src/test/java/com/graphhopper/routing/ch/PrepareContractionHierarchiesTest.java

```diff
@@ -463,18 +463,14 @@ public void testStallOnDemandViaVirtuaNode_issue1574() {
         // the weight of edge 3-1 must be such that node 2 gets stalled in the forward search via the incoming shortcut
         // at node 2 coming from 3. this happens because due to the virtual node x between 3 and 1, the spt entries
         // calculated on the query graph (using the virtual edges) use different floating point rounding / arithmetics.
-        IntsRef flags = edge31.getFlags();
-        carEncoder.getAverageSpeedEnc().setDecimal(false, flags, 22);
-        edge31.setFlags(flags);
+        edge31.set(carEncoder.getAverageSpeedEnc(), 22);
```

Comment:
This is the 'new' & right way to do it, right ? very nice!

Reply:
Yes :)